micrometer-core/src/main/java/io/micrometer/core/instrument/push/PushRegistryConfig.java

```diff
@@ -44,9 +44,15 @@ default boolean enabled() {
     }
 
     /**
+     * Return the number of threads to use with the scheduler.
+     *
+     * Note that this configuration is NOT supported.
+     *
      * @return The number of threads to use with the scheduler. The default is
      * 2 threads.
+     * @deprecated since 1.1.13
```

Comment:
```suggestion
     * @deprecated since 1.1.13 because this configuration is not used
```

Reply:
@shakuzen Thanks! I applied the suggested change in 66cfe68.